rpc_component/cli.py

```diff
@@ -242,6 +242,42 @@ def dependency(components_dir, **kwargs):
         )
 
 
+def metadata(components_dir, **kwargs):
+    metadata_dir = kwargs.pop("metadata_dir")
+    metadata_filename = "component_metadata.yml"
+    filepath = os.path.join(metadata_dir, metadata_filename)
+    new_metadata = {"artifacts": [], "dependencies": []}
+    try:
+        metadata = c_lib.load_data(filepath)
+    except yaml.parser.ParserError as e:
```

Comment:
I think it would be better to put this in the `c_lib.load_data` function.

Reply:
Fair enough.